src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -182,21 +190,24 @@ describe('NameAndPictureScreen', () => {
     expect(getByText('nameAndPicGuideCopyTitle')).toBeTruthy()
     expect(getByText('nameAndPicGuideCopyContent')).toBeTruthy()
   })
+
   it('does not render skip button when configured so', () => {
     // TODO replace route param with mock Statsig flag
```

Comment:
this comment doesn't make sense anymore, since it uses a different way of mocking statsig feature flags. please update it so it doesn't become stale